<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>441</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>128</width>
    <height>64</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Small Fonts</family>
    <pointsize>10</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Curves</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icon.png</normaloff>Icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #faf5fc;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="90,20">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="RenderArea" name="renderArea" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,2,2,2,2,0,0,0,0">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnAstroid">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>143</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Astroid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCycloid">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>143</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Cycloid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnHuygens">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>143</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Huygens</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnHypo">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>143</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Hypo Cycloid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLine">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>143</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Line</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>13</height>
             </size>
            </property>
            <property name="text">
             <string>Scale:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderScale">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>18</height>
             </size>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>19</height>
             </size>
            </property>
            <property name="text">
             <string>Interval:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinInterval">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>19</height>
             </size>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.300000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>13</height>
             </size>
            </property>
            <property name="text">
             <string>Step Count: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderStep">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>18</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>512</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderArea</class>
   <extends>QWidget</extends>
   <header>RenderArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
